com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/ScreenSpaceShadowManager.cs

```diff
@@ -203,6 +203,22 @@ void InitializeScreenSpaceShadows()
             // Directional shadow material
             s_ScreenSpaceShadowsMat = CoreUtils.CreateEngineMaterial(screenSpaceShadowsShader);
 
+            switch (m_Asset.currentPlatformRenderPipelineSettings.hdShadowInitParams.shadowFilteringQuality)
```

Comment:
Is this init function called upon changing the value in the UI?

Reply:
yes. Same as in lightloop (where we call the code in InizializeLightloop)

Reply:
Yeah, any change in something in the HDRP asset will recreate the pipeline from scratch.